mlir-tensorrt/compiler/include/mlir-tensorrt-c/Compiler/Compiler.h

```diff
@@ -60,13 +60,6 @@ typedef struct MTRT_StableHLOToExecutableOptions {
   void *ptr;
 } MTRT_StableHLOToExecutableOptions;
 
-/// A callback that allows the user to customize the metadata set for layers
-/// corresponding to each MLIR operation. The callback should invoke the
```

Comment:
Do not merge; it looks like we'll have to manually incoporate external changes into internal.